mxcube3/ui/reducers/queue.js

```diff
@@ -78,7 +78,7 @@ export default (state = initialState, action) => {
       {
         return {
           ...state,
-          rootPath: action.data.rootPath,
+          rootPath: action.data.beamlineSetup.path,
```

Comment:
is `action.data.rootPath` removed from the action data ? (since we do not use it anymore)